src/main.js

```diff
@@ -14,12 +14,24 @@ var tagline = document.querySelector('.tagline');
 var taglineOne = document.querySelector('.tagline-1');
 var taglineTwo = document.querySelector('.tagline-2');
 
-//view
+//view section
 var homeView = document.querySelector('.home-view');
 var savedView = document.querySelector('.saved-view');
 var formView = document.querySelector('.form-view');
 
+// view sub section
```

Comment:
Awesome, not sure that we'll need these, but we can delete in refactoring if necessary. Thanks!